dev/com.ibm.websphere.appserver.features/visibility/public/mpJwt-1.2/com.ibm.websphere.appserver.mpJwt-1.2.feature

```diff
@@ -15,9 +15,9 @@ Subsystem-Name: MicroProfile JSON Web Token 1.2
   com.ibm.websphere.appserver.jsonp-1.1, \
   com.ibm.websphere.appserver.httpcommons-1.0, \
   com.ibm.websphere.appserver.mpConfig-2.0, \
+  com.ibm.websphere.appserver.org.eclipse.microprofile.jwt-1.2, \
   io.openliberty.mpCompatible-4.0
 -bundles=com.ibm.ws.security.mp.jwt,\
-  io.openliberty.org.eclipse.microprofile.jwt.1.2; location:="dev/api/stable/,lib/"; mavenCoordinates="org.eclipse.microprofile.jwt:microprofile-jwt-auth-api:1.2-RC1",\
```

Comment:
Why did you delete this? Is this in the api feature?

Reply:
yes, that bundle is the API